src/lib/app/base.ts

```diff
@@ -5,6 +5,7 @@ import { logout } from './identity'
 import { showNotification } from './notification'
 import { NotificationType } from './types'
 import { persistent } from './utils'
+import type { UserRoles } from '@iota/is-shared-modules/lib/models/types/user.js'
```

Comment:
We've tried this before but this will make problems when installing it in the dashboard.

Reply:
Okay I didn't know. I will remove the shared modules then